i18n/vscode-language-pack-uk-UA/translations/extensions/ms-vscode.js-debug.i18n.json

```diff
@@ -0,0 +1,491 @@
+{
+	"": [
+		"--------------------------------------------------------------------------------------------",
+		"Copyright (c) Microsoft Corporation. All rights reserved.",
+		"Licensed under the MIT License. See License.txt in the project root for license information.",
+		"--------------------------------------------------------------------------------------------",
+		"Do not edit this file. It is machine generated."
```

Comment:
Why is a machine-generated file in git? This is not necessarily a question to the author of this PR.